.gitignore

```diff
@@ -1,5 +1,7 @@
 /node_modules
 /dist
+# compiled devtools files
+/tools/devtools/assets/*.js
```

Comment:
hmm, interesting... do you remember that you refused my very  important PR on this: https://github.com/odoo/owl/pull/1366 ...

Reply:
:sweat_smile: